src/language-js/parser-meriyah.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+const createError = require("../common/parser-create-error");
+const { hasPragma } = require("./pragma");
+const { locStart, locEnd } = require("./loc");
+const postprocess = require("./postprocess");
+
+// https://github.com/meriyah/meriyah/blob/4676f60b6c149d7082bde2c9147f9ae2359c8075/src/parser.ts#L185
+const parseOptions = {
+  // Allow module code
+  module: true,
+  // Enable stage 3 support (ESNext)
+  next: true,
+  // Enable start and end offsets to each node
+  ranges: true,
+  // Enable web compatibility
+  webcompat: true,
+  // Enable line/column location information to each node
+  loc: true,
+  // Attach raw property to each literal and identifier node
+  raw: true,
+  // Enabled directives
+  directives: true,
+  // Allow return in the global scope
+  globalReturn: true,
+  // Enable implied strict mode
+  impliedStrict: false,
+  // Enable non-standard parenthesized expression node
+  preserveParens: false,
+  // Enable lexical binding and scope tracking
+  lexical: true,
```

Comment:
What this option is enabled for?

Reply:
I wrote this several month ago, I'll take a better look on these options.